Form test uses regex that sometimes fails

See failure.

Chrome 56.0.2924 (Linux 0.0.0) amp-form Var Substitution should substitute hidden fields variables in XHR async FAILED
	AssertionError: expected 'amp--MEn6bEkgSVHw56rzwQKXztL5XRitMBLRMtRo8AfaXaLvbk-4MHRae1AE7WG_ukx' to match /amp-\w+/
	    at /tmp/688b7d6108a484955606c3168b6acfba.browserify:57087:42 <- /home/travis/build/ampproject/amphtml/extensions/amp-form/0.1/test/test-amp-form.js:1075:41

Seems like that regex needs to be made slightly more liberal.

--- a/extensions/amp-form/0.1/test/test-amp-form.js
+++ b/extensions/amp-form/0.1/test/test-amp-form.js
@@ -1055,7 +1055,7 @@ describe('amp-form', () => {
             .to.have.been.calledWith(canonicalUrlField);
         return timer.promise(10).then(() => {
           expect(ampForm.xhr_.fetch).to.be.called;
-          expect(clientIdField.value).to.match(/amp-\w+/);
+          expect(clientIdField.value).to.match(/amp-.+/);
           expect(canonicalUrlField.value).to.equal(
               'https%3A%2F%2Fexample.com%2Famps.html');
         });

